zkevm-circuits/src/witness/rw.rs

```diff
@@ -191,6 +191,275 @@ impl RwMap {
 
         rows
     }
+
+    /// Get RwMap for a chunk specified by start and end
+    pub fn from_chunked(
+        container: &operation::OperationContainer,
+        start: usize,
+        end: usize,
+    ) -> Self {
+        let mut rws = HashMap::default();
+
```

Comment:
i just tried but not possible... because `op: Operation<T: Op>` is generic, and ChatGpt says "In Rust, you cannot directly declare generic parameters on closures as you would with functions. Closures in Rust are anonymous functions and their parameter types are usually inferred by the compiler based on how they are used. "